src/worker/runner/test-runner/index.js

```diff
@@ -119,6 +120,8 @@ module.exports = class TestRunner extends Runner {
         this._browserAgent.freeBrowser(browser);
 
         if (error) {
+            filterExtraWdioFrames(error);
```

Comment:
Moved `filter` to here, because stacktrace hooks are not executed in browser runner.
That way we can filter wdio frames in only one place